src/EnergyPlus/HVACManager.cc

```diff
@@ -2761,62 +2621,60 @@ void ReportAirHeatBalance(EnergyPlusData &state)
-                            MixSenLoad(ZoneLoop) +=
-                                RefDoorMixing(ZoneLoop).VolRefDoorFlowRate(j) * AirDensity * CpAir * (thisZoneHB.MAT - zoneBHB.MAT);
-                            MixLatLoad(ZoneLoop) += RefDoorMixing(ZoneLoop).VolRefDoorFlowRate(j) * AirDensity *
-                                                    (thisZoneHB.ZoneAirHumRat - zoneBHB.ZoneAirHumRat) * H2OHtOfVap;
+                        if (refDoorMixing.VolRefDoorFlowRate(j) > 0.0) {
+                            int ZoneB = refDoorMixing.MateZonePtr(j);
+                            auto const &zoneBHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ZoneB);
+                            AirDensity = Psychrometrics::PsyRhoAirFnPbTdbW(state,
+                                                                           state.dataEnvrn->OutBaroPress,
+                                                                           (thisZoneHB.MAT + zoneBHB.MAT) / 2.0,
+                                                                           (thisZoneHB.ZoneAirHumRat + zoneBHB.ZoneAirHumRat) / 2.0,
+                                                                           std::string());
+                            CpAir = Psychrometrics::PsyCpAirFnW((thisZoneHB.ZoneAirHumRat + zoneBHB.ZoneAirHumRat) / 2.0);
+                            H2OHtOfVap = Psychrometrics::PsyHgAirFnWTdb((thisZoneHB.ZoneAirHumRat + zoneBHB.ZoneAirHumRat) / 2.0,
+                                                                        (thisZoneHB.MAT + zoneBHB.MAT) / 2.0);
+                            znAirRpt.MixVolume += refDoorMixing.VolRefDoorFlowRate(j) * TimeStepSysSec * ADSCorrectionFactor;
+                            znAirRpt.MixVdotCurDensity += refDoorMixing.VolRefDoorFlowRate(j) * ADSCorrectionFactor;
+                            znAirRpt.MixMass += refDoorMixing.VolRefDoorFlowRate(j) * AirDensity * TimeStepSysSec * ADSCorrectionFactor;
+                            znAirRpt.MixMdot += refDoorMixing.VolRefDoorFlowRate(j) * AirDensity * ADSCorrectionFactor;
+                            znAirRpt.MixVdotStdDensity +=
+                                refDoorMixing.VolRefDoorFlowRate(j) * (AirDensity / state.dataEnvrn->StdRhoAir) * ADSCorrectionFactor;
+                            MixSenLoad(ZoneLoop) += refDoorMixing.VolRefDoorFlowRate(j) * AirDensity * CpAir * (thisZoneHB.MAT - zoneBHB.MAT);
+                            MixLatLoad(ZoneLoop) +=
+                                refDoorMixing.VolRefDoorFlowRate(j) * AirDensity * (thisZoneHB.ZoneAirHumRat - zoneBHB.ZoneAirHumRat) * H2OHtOfVap;
                         } // flow > 0
                     }     // J-1, numref connections
                 }         // zone A (zoneptr = zoneloop)
                 for (int ZoneA = 1; ZoneA <= (ZoneLoop - 1); ++ZoneA) {
-                    auto &zoneAHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ZoneA);
```

Comment:
[src/EnergyPlus/HVACManager.cc:2795]:(style),[constVariable],Variable 'zoneAHB' can be declared as reference to const